### ISSUE TITLE
"Manage folders" action not responding

### ISSUE THREAD

**Issue by suprune20** [messagerie-melanie2/Roundcube-Plugin-JQuery-Mobile#4]
roundcube: 1.3.6, imap backend: courier-imap
"Manage folders" ( https://webmail-rc.bsuir.by/?_task=settings&_action=folders ) fails in mobile view: no response. Cromium debuggers output:

jquery.min.js?s=1523445242:36 Uncaught Error: No label found for checkboxradio widget
    at Function.error (jquery.min.js?s=1523445242:36)
    at t.(/anonymous function).(anonymous function)._getCreateOptions (https://webmail-rc.bsuir.by/plugins/jqueryui/js/jquery-ui.min.js?s=1523445226:9:29101)
    at t.(/anonymous function).(anonymous function)._getCreateOptions (https://webmail-rc.bsuir.by/plugins/jqueryui/js/jquery-ui.min.js?s=1523445226:6:4499)
    at t.(/anonymous function).(anonymous function)._createWidget (https://webmail-rc.bsuir.by/plugins/jqueryui/js/jquery-ui.min.js?s=1523445226:6:6580)
    at new t.(/anonymous function).(anonymous function) (https://webmail-rc.bsuir.by/plugins/jqueryui/js/jquery-ui.min.js?s=1523445226:6:4063)
    at HTMLInputElement.<anonymous> (jquery-ui.min.js?s=1523445226:6)
    at Function.each (jquery.min.js?s=1523445242:36)
    at r.fn.init.each (jquery.min.js?s=1523445242:36)
    at r.fn.init.t.fn.(/anonymous function) [as checkboxradio] (https://webmail-rc.bsuir.by/plugins/jqueryui/js/jquery-ui.min.js?s=1523445226:6:5750)
    at r.fn.init.enhanceWithin (jquery.mobile-1.4.5.….js?s=1529053603:13)

**Comment by tpayen** [comment]
Do you still have the problem with roundcube 1.3.8 ?

**Comment by suprune20** [comment]
Yes, the problem is still in roundcube 1.3.8. Cromium debugger output:

jquery.min.js?s=1540293145:36 Uncaught Error: No label found for checkboxradio widget
    at Function.error (jquery.min.js?s=1540293145:36)
    at t.(/mail-new/anonymous function).(anonymous function)._getCreateOptions (https://round.bsuir.by/mail-new/plugins/jqueryui/js/jquery-ui.min.js?s=1540293134:9:29101)
    at t.(/mail-new/anonymous function).(anonymous function)._getCreateOptions (https://round.bsuir.by/mail-new/plugins/jqueryui/js/jquery-ui.min.js?s=1540293134:6:4499)
    at t.(/mail-new/anonymous function).(anonymous function)._createWidget (https://round.bsuir.by/mail-new/plugins/jqueryui/js/jquery-ui.min.js?s=1540293134:6:6580)
    at new t.(/mail-new/anonymous function).(anonymous function) (https://round.bsuir.by/mail-new/plugins/jqueryui/js/jquery-ui.min.js?s=1540293134:6:4063)
    at HTMLInputElement.<anonymous> (jquery-ui.min.js?s=1540293134:6)
    at Function.each (jquery.min.js?s=1540293145:36)
    at r.fn.init.each (jquery.min.js?s=1540293145:36)
    at r.fn.init.t.fn.(/mail-new/anonymous function) [as checkboxradio] (https://round.bsuir.by/mail-new/plugins/jqueryui/js/jquery-ui.min.js?s=1540293134:6:5750)
    at r.fn.init.enhanceWithin (jquery.mobile-1.4.5.….js?s=1543825116:13)
﻿

**Comment by HexDestroyer** [comment]
I have this problem too.
It's a jqueryui feature https://bugs.jqueryui.com/ticket/15138#no1
The quick workaround I've found is to modify jquery.mobile-1.4.5.js
Row 600:

```
for ( index in widgetElements ) {
    widgetElements[ index ][ index ]();
}
```

to
```
for ( index in widgetElements ) {
    try {
        widgetElements[ index ][ index ]();
    }
    catch(error) {
        console.error(error);
    }
}
```
The error will still be there, but now it will not get stuck

**Comment by vrubiella** [comment]
I have this problem too. (roudncube 1.3.8)
Thks for js workaround prefents freeze view.
jquery issue is closed:

> That markup is not accessible and is therefore not supported. You must provide the < label > yourself.

I supose that the correct solution is fix generated html no?
